src/es/components/atoms/iframe/Iframe.js

```diff
@@ -62,24 +74,38 @@ export default class Iframe extends Intersection() {
    * @return {void}
    */
   renderCSS () {
-    this.css = /* css */`
+    this.css = /* css */ `
       :host {
         line-height: 0;
       }
       :host, :host > iframe {
-        ${this.iframe && this.iframe.getAttribute('width') && !this.iframe.getAttribute('width').includes('%') && this.iframe.getAttribute('height') && !this.iframe.getAttribute('height').includes('%')
-          ? `aspect-ratio: ${this.iframe.getAttribute('width')} / ${this.iframe.getAttribute('height')};`
-          // @ts-ignore
-          : console.warn('This component requires an Iframe with fix/absolute width and height values: ', this) || ''
+        ${
+          this.iframe &&
+          this.iframe.getAttribute('width') &&
+          !this.iframe.getAttribute('width').includes('%') &&
+          this.iframe.getAttribute('height') &&
+          !this.iframe.getAttribute('height').includes('%')
+            ? `aspect-ratio: ${this.iframe.getAttribute(
+                'width'
+              )} / ${this.iframe.getAttribute('height')};`
+            : // @ts-ignore
+              console.warn(
+                'This component requires an Iframe with fix/absolute width and height values: ',
+                this
+              ) || ''
         }
         width: 100%;
         height: auto;
-        ${this.hasAttribute('background-color')
-          ? `background-color: ${this.getAttribute('background-color')};`
-          : ''
+        ${
+          this.hasAttribute('background-color')
+            ? `background-color: ${this.getAttribute('background-color')};`
+            : ''
         }
         max-height: var(--max-height, 75vh);
       }
+      :host([keep-aspect-ratio]), :host([keep-aspect-ratio]) > iframe {
+        max-height: max-content;
+      }
```

Comment:
This part is new. If set, the max-height goes from 75vh (default) to max-content, to display youtube videos and keep the aspect ratio. Otherwise it will be displayed with margin on left and right side.